tuning_docs/tensile_tuning.tex

```diff
@@ -579,61 +637,97 @@ \subsubsection{The Tensile Logic File}
+
+\section{Integrating new libraries in rocBLAS}
+Once we have library logic files for new sizes, we need to merge and update rocBLAS libraries. This a two-fold process: 
+\begin{itemize}
+\item First, we need to confirm that the new logic file is generating the correct results and the performance of the new libraries is more that the old library. We use rocBLAS-bench tool for this part. See the next section. 
+\item Merging the new logic file into rocBLAS using the merge tool (merge.py in Tensile). See section \ref{sec:merging}.
+\end{itemize}
+
+
+
+\subsection{rocBLAS-bench tool}
+
+\label{sec:bench}
+
+\noindent
+rocBLAS-bench is a tool used to perform benchmarks on the problem sizes which are being measured and to verify the correctness of the new libraries. While it is designed to measure performance of the full set of rocBLAS library functions, our interest is in the level 3 gemm functions which---in rocBLAS---are calls into the Tensile library. 
 
-rocblas-bench is a tool used to perform benchmarks on the problem sizes which are being measured. While it is designed to measure performance of the full set of rocBLAS library functions, our interest is in the level 3 gemm functions which---in rocBLAS---are calls into the Tensile library.
 
 The git repository for rocBLAS is found at \url{https://github.com/ROCmSoftwarePlatform/rocBLAS}. Either the develop or master branch version may be used for testing.
 
 \begin{lstlisting}[language=bash]
-git clone -b master|develop https://github.com/ROCmSoftwarePlatform/rocBLAS.git
+$ git clone -b master|develop https://github.com/ROCmSoftwarePlatform/rocBLAS.git
 \end{lstlisting}
 
-\noindent
-The library can be installed using the included installation script:
+To use rocBLAS-bench we need to install it locally. The library can be installed using the included installation script (the installation may last quite some time.):
 
 \begin{lstlisting}[language=bash]
-cd rocBLAS
-./install.sh -d -c
+$ cd rocBLAS
+$ ./install.sh -h
+$ ./install.sh -d -c -a <architecture> --cmake_install
 \end{lstlisting}
 
 \noindent
-where the c option builds the client applications, which includes rocblas-bench, and the d option will build and install the dependencies (the d option only need to be included once on each system that rocblas is being build).
+where the c option builds the client applications, which includes rocBLAS-bench, and the d option will build and install the dependencies (the d option only need to be included once on each system that rocBLAS is being build). -a refers to the GPU architecture. For example: gfx906:xnack-, gfx908:xnack-. Without this flag, rocBLAS installs/tunes for all architectures, and installation lasts even longer. To skip rocBLAS Tensile use -n flag (You cannot benchmark GEMM then). The cmake flag allows the rocBLAS to install cmake if it is older than required.
```

Comment:
```suggestion
where the c option builds the client applications, which includes rocBLAS-bench, the d option builds and installs the dependencies (only needed to be included once on each system that rocBLAS is being built), and the a option specifies the GPU architecture e.g. gfx906:xnack-, gfx908:xnack-. Without this flag, rocBLAS installs for all architectures, and installation lasts even longer. The cmake flag allows the rocBLAS to install cmake if it is older than required.
```

Reply:
Cleaning up a bit. Removed the bit about the -n flag; I think this just adds confusion since this is a Tensile doc and you would only use rocBLAS if it was built with Tensile.